### PR TITLE
Added check-elasticache-failover.rb

### PR DESCRIPTION
**Pull request by y13i**
This checks if specified ElastiCache node is `primary` state.

## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [ ] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [x] RuboCop passes

- [ ] Existing tests pass 

#### New Plugins

- [x] Tests

- [x] Add the plugin to the README

- [x] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

File: bin/check-elasticache-failover.rb [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+    [:profile, :region].each do |option|
+      h.update(option => config[option]) if config[option]
+    end
+
+    h.update(region: own_region) if h[:region].nil?
+    h
+  end
+
+  def own_region
```
Comment (sstarcher) [inline bin/check-elasticache-failover.rb]: could you place this in the lib/sensu-plugins-aws/common.rb

File: bin/check-elasticache-failover.rb [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    end
+  end
+
+  private
+
+  def elasticache
+    @elasticache ||= Aws::ElastiCache::Resource.new(aws_configuration)
+  end
+
+  def aws_configuration
```
Comment (sstarcher) [inline bin/check-elasticache-failover.rb]: Remove this as it's replaced by lib/sensu-plugins-aws/common.rb

**Comment by y13i** [comment]
@sstarcher I modified the code to use `include Common`. 764c961

**Comment by sstarcher** [comment]
@y13i looks good did you mean to fully remove the "own_region" functionality?

**Comment by y13i** [comment]
@sstarcher 

> did you mean to fully remove the "own_region" functionality?

I think it may be useful sometimes, but not necessary.

**Pull request merged**